sw/device/tests/sim_dv/sysrst_ctrl_ec_rst_l_test.c

```diff
@@ -140,12 +142,12 @@ static void check_combo_reset(void) {
 // indicate that the testbench is ready to proceed with the
 // next phase of the test.
```

Comment:
nit: Since we are updating `wait_next_test_phase`, maybe we can also replace inline comments for consistency
```
/**
 * Waits for the kTestPhase variable to be changed by a backdoor overwrite
 * from the testbench in chip_sw_sysrst_ctrl_ec_rst_l_vseq.sv. This will
 * indicate that the testbench is ready to proceed with the
 * next phase of the test.
 */
```

Reply:
Done